README.md

```diff
@@ -384,19 +384,23 @@ amadeus.booking.hotelBookings.post(
 
 // Points of Interest
 // What are the popular places in Barcelona (based a geo location and a radius)
-amadeus.referenceData.locations.pointsOfInterest.get({
+amadeus.referenceData.locations.pointsOfInterest().get({
```

Comment:
Any particular reason to add () there? I am not an expert in Node but on a similar API we didn't. Is it for visibility?

<img width="478" alt="image" src="https://user-images.githubusercontent.com/713173/79279277-f0a4de00-7ead-11ea-80f6-69f77c29b54f.png">

Reply:
In the example you posted - location() is a function, while locations is an object. Hence, it is possible to avoid () in this case.

Reply:
Not sure I fully understood the answer. The case I put in the screenshot is very similar to the POIs, we have one endpoint that allows to do a search and an endpoint to retrieve a specific location. @alnacle stop me if I am wrong but I think this is how we manage this case:
- Singular for the retrieve
- Plural for the searches

If it is possible to avoid the () please remove them to keep consistency with other API examples

Reply:
Sorry, I didn't see this thread before and I answered at the bottom.

Yes, the idea is what @anthonyroux suggests: singular for retrieve and plural for searches.

`location()` is a function implemented in `reference_data.js` file that returns a `Location` object (singular), which calls the `get()` method underneath. This is the way we implemented the call to the retriever like `referenData.location('FOO').get()` rather than`referenData.location.get('FOO')`